raft.c

```diff
@@ -1339,6 +1411,91 @@ void RaftReqHandleQueue(uv_async_t *handle)
  * Implementation of specific request types.
  */
 
+static void handleTransferLeaderComplete(raft_server_t *raft, raft_transfer_state_e state)
+{
+    raft_node_id_t transfer_id = raft_get_transfer_leader(raft);
+    raft_node_id_t leader = raft_get_leader_id(raft);
```

Comment:
Unused variable, leftover?

Reply:
sort of, I wasn't sure if it should be included in the unexpected leader election message